tools/build_fuchsia_sdk.sh

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
+# Copyright 2023 syzkaller project authors. All rights reserved.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+# First follow the tutorial at https://fuchsia.dev/fuchsia-src/get-started/build_fuchsia
+# Set the following env variables before executing this script:
+# SYZKALLER -- path to the syzkaller checkout.
+# FUCHSIA -- path to the fuchsia checkout.
+# SDK -- where the generated sdk should be stored.
+
+
+if [ -z "${SYZKALLER}" ]; then
+  echo "The SYZKALLER environment variable is not set."
+  exit 1
+fi
+
+if [ -z "${FUCHSIA}" ]; then
+  echo "The FUCHSIA environment variable is not set."
+  exit 1
+fi
+
+if [ -z "${SDK}" ]; then
+  echo "The SDK environment variable is not set."
+  exit 1
+fi
+
+# Copy generic SDK files.
+
+function copy_dir() {
+    FROM=${1}
+    TO=${2}
+    FLAGS=${3}
+    if [ -z "$FLAGS" ]
+    then
+      FLAGS="-rL"
+    fi
+    mkdir -p ${TO}
+    echo "Copying ${FLAGS} ${FROM}/. to ${TO}"
+    cp ${FLAGS} ${FROM}/. ${TO}
+}
+
+mkdir -p $SDK
+
+for folder in sdk/lib src/lib zircon; do
+  copy_dir ${FUCHSIA}/${folder} ${SDK}/${folder}
+done
+copy_dir ${FUCHSIA}/prebuilt/third_party/clang ${SDK}/prebuilt/third_party/clang "-r --preserve=links"
+
+# Build and copy arm64 files.
+
+cd ${FUCHSIA}
+
+fx --dir "out/arm64" set core.arm64 \
```

Comment:
At some point it would be ideal to de-dupe this with the similar code in https://github.com/google/syzkaller/blob/master/docs/fuchsia/setup.sh#L72 but will address as a followup.